src/zodbbrowser/search.py

```diff
@@ -0,0 +1,96 @@
+import cPickle
+import io
+import logging
+import optparse
+import sys
+import types
+
+import ZODB.broken
+from zodbbrowser.standalone import open_database
+
+
+def is_broken(symb):
+    """Return true if the given symbol is broken.
+    """
+    return (isinstance(symb, types.TypeType) and
+            ZODB.broken.Broken in symb.__mro__)
+
+
+class Search(object):
+
+    def __init__(self, classes=[]):
+        self.classes = classes
+
+    def validate(self, cls_info, cls):
+        fullname = '.'.join(cls_info)
+        if is_broken(cls):
+            print 'Broken class {}'.format(fullname)
+        if self.classes and fullname in self.classes:
+            print 'Instance of {} found'.format(fullname)
```

Comment:
Eventually I'll want a search page in the web app.  It'd be neat if printing the results didn't have to be hardcoded.